src/app/features/auth/services/auth.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from "@angular/common/http";
+import { User } from '../../../shared/models/user';
+import { Observable, shareReplay, tap } from "rxjs";
+import { Token } from '../../../shared/models/token';
+import { StorageService } from "../../../core/services/storage.service";
+import { environment } from '@src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+  private readonly baseUrl: string = `${environment.apiUrl}`;
+  private readonly httpOptions = {
+    headers: new HttpHeaders({ 'Content-Type': 'application/json' })
+  };
+
+  constructor(private readonly http: HttpClient, private readonly localStorage: StorageService) {
+  }
+
+  public login(username: string, password: string, isRemembered = false): Observable<Token> {
+    return this.http.get<Token>(
+      this.baseUrl + `auth/token?email=${username}&password=${password}`
```

Comment:
Query params should not be added manually. `HttpClient.get` method has second `options` argument, where you can pass params as object.